pkgs/tools/inputmethods/fcitx5/fcitx5-chinese-addons.nix

```diff
@@ -0,0 +1,86 @@
+{ stdenv
+, mkDerivation
+, fetchurl
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, boost
+, libime
+, fcitx5
+, fcitx5-qt
+, fcitx5-lua
+, qtwebengine
+, opencc
+, curl
+, luaSupport ? true
+}:
+
+let
+  pyStrokeVer = "20121124";
+  pyStroke = fetchurl {
+    url = "http://download.fcitx-im.org/data/py_stroke-20121124.tar.gz";
+    sha256 = "0j72ckmza5d671n2zg0psg7z9iils4gyxz7jgkk54fd4pyljiccf";
+  };
+  pyTableVer = "20121124";
+  pyTable = fetchurl {
+    url = "http://download.fcitx-im.org/data/py_table-20121124.tar.gz";
+    sha256 = "011cg7wssssm6hm564cwkrrnck2zj5rxi7p9z5akvhg6gp4nl522";
+  };
+in
+
+mkDerivation rec {
+  pname = "fcitx5-chinese-addons";
+  version = "5.0.1";
+
+  src = fetchFromGitHub {
+    owner = "fcitx";
+    repo = "fcitx5-chinese-addons";
+    rev = version;
+    sha256 = "sha256-udPgyKfIudzS4aXXYiDTjKlcRwm27ity5paCw0y73Sk=";
+  };
+
+  cmakeFlags = [
+    "-DUSE_WEBKIT=off"
+    "-DCMAKE_INSTALL_DATADIR=${placeholder "out"}/share"
+    "-DFEM_LIB_INSTALL_DIR=${placeholder "out"}/lib"
```

Comment:
This shouldn't be set, its an temp variable in cmake function.

Reply:
Yeah, they are not necessary after recent updates.